src/Services/Export/WB.Services.Export.Host/Controllers/TenantController.cs

```diff
@@ -40,15 +51,50 @@ public async Task<ActionResult> StopTenant([FromQuery] string? tenant = null)
                 tenant = context.Tenant.Name;
             }
             
-            if (string.IsNullOrWhiteSpace(tenant)) return BadRequest("No tenant specified");
+            if (string.IsNullOrWhiteSpace(tenant)) 
+                return BadRequest("No tenant specified");
+            
+            this.logger.LogCritical("Export service tenant {tenant} data deleted due to the request", tenant);
+            dropTenantStatuses[tenant] = StopTenantStatus.Removing;
+            try
+            {
+                await this.questionnaireSchemaGenerator.DropTenantSchemaAsync(tenant);
+                var jobsCount = await this.archiver.ArchiveJobs(tenant);
+                dropTenantStatuses[tenant] = StopTenantStatus.Removed;
+                return Ok(new
+                {
+                    archivedJobs = jobsCount
+                });
+            }
+            catch (Exception e)
+            {
+                this.logger.LogCritical(e, "Fail to drop tenant {tenant}", tenant);
+                dropTenantStatuses[tenant] = StopTenantStatus.Error;
+                throw;
+            }
+        }
+
+        
+        [HttpGet]
+        [Route("api/v1/statusDeleteTenant")]
+        public ActionResult StatusStopTenant([FromQuery] string? tenant = null)

```

Comment:
Please rename StatusStopTenant to DroppingTenantStatus (or GetDroppingTenantStatus).
Please rename StopTenant -> DropTenant